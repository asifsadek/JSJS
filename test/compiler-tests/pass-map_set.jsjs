val a = { "foo" : "bar"};
print(set(a, "foo1", "bar1"));

val b = { 1 : [1, 2, 3] };
print(set(b, 2, [1, 4, 9]));

val c = { true : 2, 
          false : 3 };
print(set(c, true, 4));

val d = { 1 : { "foo" : 2 }};
print(set(d, 2, {"bar": 4}));
