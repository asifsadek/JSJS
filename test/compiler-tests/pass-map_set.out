Map { "foo": "bar", "foo1": "bar1" }
Map { "1": List [ 1, 2, 3 ], "2": List [ 1, 4, 9 ] }
Map { "true": 4, "false": 3 }
Map { "1": Map { "foo": 2 }, "2": Map { "bar": 4 } }
